Week2/homework/map_filter.js

```diff
@@ -2,4 +2,15 @@
 
 const numbers = [1, 2, 3, 4];
 
-// Add your code here
+
+const newNumber = numbers.filter( number => number % 2 !== 0);
+console.log(newNumber);
```

Comment:
Please remove the extra `console.log` statements. Since they are only used for debugging by a developer, they should be removed if you think the code is ready to be pushed. 🙂